app/controllers/medicaid/expenses_student_loan_member_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Medicaid
+  # rubocop:disable LineLength
```

Comment:
in other controllers we've just breaking these up onto 2 lines rather than disabling rubocop

Reply:
@jessieay @hartsick how should I break this up into 2 lines?

```ruby
class ExpensesStudentLoanMemberController < Medicaid::ManyMemberStepsController
```

I don't know if I've seen classes that have had to do that yet so I want to follow y'all's lead.

This ok?

```ruby
class ExpensesStudentLoanMemberController <
  Medicaid::ManyMemberStepsController
```